lib/backend_common/backend_common/notifications.py

```diff
@@ -17,3 +26,69 @@
 URGENCY_LEVELS = [
     'LOW', 'NORMAL', 'HIGH',
 ]
+
+
+def get_current_app_credentials() -> dict:
+    return {
+        'id': current_app.config.get('TASKCLUSTER_CLIENT_ID'),
+        'key': current_app.config.get('TASKCLUSTER_ACCESS_TOKEN'),
+        'algorithm': 'sha256',
+    }
+
+
+def generate_random_uid() -> str:
+    return ''.join(random.choice(string.ascii_letters + string.digits) for _ in range(32))
+
+
+def verify_policy_structure(policy: dict) -> None:
+    if any(key not in policy for key in ['frequency', 'identity', 'start_timestamp', 'stop_timestamp', 'urgency']):
+        raise KeyError('Policy missing required key')
+
+    if any(freq_key not in policy['frequency'] for freq_key in ['days', 'hours', 'minutes']):
+        raise KeyError('Policy frequency missing required key')
+
+
+def create_new_nagbot_message(message: str, short_message: str, deadline: datetime, policies: List[dict], uid: str=None) -> str:
```

Comment:
instead of using the name ``create_new_nagbog_message`` I think more descriptive would be ``shedule_new_notification``.